src/uct/api/uct_def.h

```diff
@@ -289,12 +291,16 @@ typedef void (*uct_unpack_callback_t)(void *arg, const void *data, size_t length
  *
  * @retval UCS_OK         - the server will accept the connection request from
  *                          the client.
+ * @retval UCS_INPROGRESS - the user should accept or reject the request with
```

Comment:
since this is not fast path, maybe easier that user could call accept/reject from the callback? to simplify the UCT API usage

Reply:
we need to keep `id` untill app calls `ucp_ep_create` or `ucp_listener_reject`

Reply:
yes, i mean the callback will return void and use ep_create/reject

Reply:
then user must call accept or reject from callback or later but it's always his responsibility, right?

Reply:
yes

Reply:
ok